spec/requests/site_controller_spec.rb

```diff
@@ -29,7 +29,7 @@
       expect(json["header_primary_color"]).to eq("333333")
       expect(json["header_background_color"]).to eq("ffffff")
       expect(json["login_required"]).to eq(true)
-      expect(json["discourse_discover_enrolled"]).to eq(true)
+      expect(json["discourse_discover"]["enrolled"]).to eq(true)
```

Comment:
It may be best to have the same structure on both requests, less confusing. 

`/site/statistics.json` returns: 

```
"discourse_discover_enrolled"=>true,
"discourse_discover_logo_url"=> URL,
"discourse_discover_locale"=>"en"
```

Can we keep the same for `/site/basic-info.json`? Also... do we need to expose this info in both requests? We could limit it to one only.

Reply:
Maybe we can only use the `basic-info`. endpoint. I used the way we get the forum title and description. We get this from both about.json and basic-info.json endpoints. I will check the possibilities later.

Reply:
Sounds good, please @ me when this is ready for another review, thanks!